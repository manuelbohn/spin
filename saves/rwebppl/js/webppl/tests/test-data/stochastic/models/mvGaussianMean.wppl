var model = function() {
  var mu = multivariateGaussian(Vector([0]), Vector([4]));
  factor(MultivariateGaussian({mu: mu, cov: Vector([1])}).score(Vector([5.5])));
  factor(MultivariateGaussian({mu: mu, cov: Vector([1])}).score(Vector([6.5])));
  return T.get(mu, 0);
};
