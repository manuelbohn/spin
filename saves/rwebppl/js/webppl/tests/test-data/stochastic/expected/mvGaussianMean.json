{
  "mean": 5.333333,
  "std": 0.666667
}
